### PR TITLE
Fix two new coverity issues

### PR DESCRIPTION
**Pull request by bluca**

### CODE REVIEW

File: src/libsystemd/sd-journal/journal-vacuum.c [hunk 1]
```diff
@@ -151,7 +151,8 @@ int journal_directory_vacuum(
 
                 unsigned long long seqnum = 0, realtime;
                 _cleanup_free_ char *p = NULL;
-                sd_id128_t seqnum_id;
+                /* If the journal file is archived, seqnum_id won't be initialised before use, so set to 0 */
+                sd_id128_t seqnum_id = SD_ID128_MAKE(00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00);
```
Comment (poettering) [inline src/libsystemd/sd-journal/journal-vacuum.c]: SD_ID128_NULL

Comment (poettering) [reply src/libsystemd/sd-journal/journal-vacuum.c]: maybe move the initialization with SD_ID128_NULL into the if block that handles archived journal files though. After all for non-archived ones this initialization is unnecessary

Comment (bluca) [reply src/libsystemd/sd-journal/journal-vacuum.c]: I knew there was something already. Went and checked - completely missed it anyway :-D

**Comment by yuwata** [comment]
LGTM.

File: src/libsystemd/sd-journal/journal-vacuum.c (hunk 1 above)
Comment (bluca) [reply src/libsystemd/sd-journal/journal-vacuum.c]: moved down and used SD_ID128_NULL

**Pull request merged**